src/lib/components/tablev2/SingleSelectableContent.js

```diff
@@ -117,7 +122,18 @@ export default function SingleSelectableContent({
 
             return (
               <div {...cellProps} className="td">
-                {cell.render('Cell')}
+                {
+                  // if cell.column.cell is defined that means we have a custom cell renderer
+                  // otherwise it should be just a string and we can use the constrainted text component
+                  !cell.column.cell && typeof cell.value === 'string' ? (
+                    <ConstrainedText
+                      text={cell.value}
+                      tooltipPlacement={index === 0 ? 'bottom' : 'top'}
```

Comment:
Probably an issue with the tooltip here too. 
![image](https://user-images.githubusercontent.com/18453133/146773246-9edda4b7-75b4-4d74-b349-1b133c26bcd2.png)